imls-frontend/src/views/PageState.vue

```diff
@@ -18,33 +25,50 @@ export default {
     return {
       store,
       fetchCount: null,
-      fetchError: {},
-      fetchedData: [],
+      fetchError: null,
+      fetchedLibraries: [],
       isLoading: false,
     }
   },
+  computed: {
+    stateName () {
+      return this.store.states[this.stateInitials]
+    },
+    breadcrumbs () {
+      return [
+         { 
+          name: "All States",
+          link: "/" 
+        },
+        { 
+          name: this.stateName,
+          link: `/system/${this.stateInitials}` 
+        }
+      ]
+    }
+  },
   watch: {
     stateInitials(newVal, oldVal) {
       if (newVal !== oldVal) {
-        this.fetchData();
+        this.fetchLibraries();
       }
     }
   },
   async beforeMount() {
-    await this.fetchData();
+    await this.fetchLibraries();
   },
   methods: {
-    async fetchData() {
+    async fetchLibraries() {
       if (this.stateInitials.length !== 0) {
         this.isLoading = true;
         try {
           const response = await fetch(`${store.backendBaseUrl}${store.backendPaths.getAllSystemsByStateInitials}?_state_code=${this.stateInitials}`);
-          if (await !response.ok) {
-            throw new Error(response.status);
-          }
-          this.fetchedData = await response.json();
+          const parsedResponse = await response.json();
+
+          // the API currently returns null instead of an empty array on no matches
```

Comment:
@anagradova I was wondering if, for the endpoints that return zero, one, or more libraries in a response, if the 0 returned results case could return an empty array instead of `null` in the response body. What do you think?

Reply:
If zero returned results is not an error I agree that an empty array is the correct return structure. That way your downstream code to these methods don't require any special handling of the zero case.

Reply:
I can ensure that the endpoint will return an empty array if there are no results for the searches